app/decorators/event_decorator.rb

```diff
@@ -29,20 +29,17 @@ def format_self_submit_attendance(user)
   end
 
   def valid_event_types(member)
-    if member.position == 'secretary'
+    position = member.position
+    if position == Position.first
       EventType.all
     else
-      editable_event_types = EventType.where(name: member.position)
-      if member.position == 'Risk Manager'
-        editable_event_types += EventType.where(name: 'Ritual Review')
-      end
-      editable_event_types
+      position.event_type
     end
   end
 
   def format_edit_link(member)
-    if member.position == User.secretary ||
-       member.position == object.event_type.name
+    if member.position == Position.first ||
```

Comment:
How are you going to make a guarantee about what the first record in that table is?